swift.go

```diff
@@ -494,7 +494,8 @@ func (c *Connection) authenticate(ctx context.Context) (err error) {
 		}
 	}
 
-	retries := 1
+	shouldReattempt := c.Auth.ShouldReattemptAuth()
```

Comment:
This doesn't ever seem to be used?

Reply:
> This doesn't ever seem to be used?

It's checked on line 523 and assigned false on line 524, no?